### ISSUE TITLE
Training issue with and without Conv2DNormalization layer

### ISSUE THREAD

**Issue by sigmoidx** [oarriaga/SSD-keras#7]
Hi,

I'm training two different setups of SSD-keras you uploaded here on two different workstations simultaneously and found something very strange.

Here are the setups I prepared:
Setup.1: Vanilla SSD-keras (it comes with Conv2DNormalization layer after conv4_3)
Setup.2: Modded SSD-keras (it comes without Conv2DNormalization layer after conv4_3; replaced with conv4_3)

And I'm having about 138 epochs on both and here are the results:

Setup.1: loss 6.9733, val_loss: 7.0312
Setup.2: loss 4.2583, val_loss: 5.2805

As long as I know the Conv2DNormalization layer is supposed to improve the mAP and reduce loss as mentioned in the paper.

Could you explain why the setup.2 has less loss and val_loss?
Thanks,
